### PR TITLE
Fish key reader

### PR DESCRIPTION
**Pull request by krader1961**

### CODE REVIEW

File: doc_src/fish_key_reader.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+\section fish_key_reader fish_key_reader - explore what characters keyboard keys send
+
+\subsection fish_key_reader-synopsis Synopsis
+\fish{synopsis}
+fish_key_reader
+\endfish
+
+\subsection fish_indent-description Description
```
Comment (faho) [inline doc_src/fish_key_reader.txt]: Copy-paste error, should be fish_key_reader-description.

**Comment by faho** [comment]
The obvious prior art to consider here is bash's ctrl-v binding. You press it, then you press some keycombination and it shows the sequence.

It might be nice to allow something like that, to allow invoking this for one sequence.

File: doc_src/fish_key_reader.txt (hunk 1 above)
Comment (krader1961) [inline doc_src/fish_key_reader.txt]: Fixed

**Comment by krader1961** [comment]
Interesting idea regarding a "single shot" mode. Perhaps that should be the default with a `--continuous` option (or some such) for the current behavior.

**Comment by faho** [comment]
Sounds good.

**Comment by krader1961** [comment]
Merged as b055b8440c87c69e7f92a81f114ee77f48940f66

**Comment by floam** [comment]
This is #2991 and fish_key_reader was merged into mater and Integration_2.3.1.

**Pull request closed without merging**